applications/nrf5340_audio/prj.conf

```diff
@@ -8,7 +8,7 @@
 CONFIG_NRF5340_AUDIO=y
 CONFIG_REBOOT=y
 CONFIG_MAIN_THREAD_PRIORITY=10
-CONFIG_MAIN_STACK_SIZE=1940
+CONFIG_MAIN_STACK_SIZE=2800
```

Comment:
What is the reason why we need to increase the stack size?
Now the stack size is larger even if we do not include the DFU functionality, right?

Reply:
This might be leftover when debugging sdcard over SPI.
I'll remove it.